lib/wasmex.ex

```diff
@@ -104,7 +103,7 @@ defmodule Wasmex do
       e in RuntimeError -> {false, e.message}
     end
 
-    Wasmex.Native.namespace_receive_callback_result(token, success, [return_value])
+    :ok = Wasmex.Native.namespace_receive_callback_result(token, success, [return_value])
```

Comment:
this throws if we provide a wrong signature, maybe there are better ways to handle this error? throwing a proper error maybe instead of this "does not match :ok" error 🤔

Reply:
I do think it eventually needs a better error, but I think the match error is fine for now.